Qitom/python/pythonItom.cpp

```diff
@@ -181,32 +181,19 @@ PyObject* PythonItom::PyLog(PyObject* /*pSelf*/, PyObject* pArgs)
 {
     ito::RetVal retVal(ito::retOk);
 
-    if (PyTuple_Size(pArgs) != 1)
+    const char* text = NULL;
+    if (!PyArg_ParseTuple(pArgs, "s", &text))
     {
-        retVal += ito::RetVal(ito::retError, 0, "Wrong number of arguments");
```

Comment:
it would be just sufficient to write something like this:

`const char* text = nullptr;

if  (!PyArg_ParseTuple(pArgs, "s", &text)
{
    // argument parsing error. The error message is already set by PyArg_ParseTuple.
    return nullptr;
}

ito::RetVal retVal;`


a specific check for the tuple size is not necessary. this is all done in PyArg_ParseTuple.